src/modules/tenant/api.js

```diff
@@ -1,9 +1,16 @@
 export default class Api {
-    constructor(axios) {
-        this.Axios = axios;
-    }
+	constructor(axios) {
+		this.Axios = axios;
+	}
 
-    get_tenants() {
-        return this.Axios.get('/tenant')
-    }
+	get_tenants() {
+		return this.Axios.get('/tenant')
+	}
+
+	get_user_access(tenant, access_token) {
+		return this.Axios.get(
+			"/rbac/" + tenant + "/tenant:access",
```

Comment:
This needs to be generalized into auth module as `verify_access([tenant], [resource])` - resource is "tenant:access".

The access token should be handled internally, not to be exposed to the application.

Tenant module may provide this convenience method to `verify_access_to_tenant()`.